3.2/root/usr/share/container-scripts/mongodb/common.sh

```diff
@@ -259,3 +259,59 @@ function setup_keyfile() {
 function info() {
   printf "=> [%s] %s\n" "$(date +'%a %b %d %T')" "$*"
 }
+
+# insert_and_wait_for_replication insert data in host and wait all replset members
```

Comment:
ok, i didn't appreciate that you needed the script content inside the image itself, so that makes sense.

i'd still suggest either a separate file, or at least some comments on the functions indicating what they are used for/that they are for test purposes only.